spec/features/admins/merchants/index_spec.rb

```diff
@@ -108,5 +110,48 @@
         end
       end
     end 
+
+    describe 'create a new merchant' do 
+      it 'has a link to a form to create a new merchant' do 
+        visit admin_merchants_path
+
+        click_link("Create a New Merchant")
+
+        expect(current_path).to eq new_admin_merchant_path
+
+        expect(page).to have_field 'Name'
+        expect(page).to have_button 'Create Merchant'
+      end
+
+      it 'will show the new merchant on the admin merchants index page once form is submitted' do 
+        visit admin_merchants_path
+
+        click_link("Create a New Merchant")
+
+        fill_in 'Name', with: 'Betty Draper'
+        click_button 'Create Merchant'
+        
+        betty_merchant = Merchant.last
+        expect(current_path).to eq admin_merchants_path
+        expect(page).to have_content(betty_merchant.name)
```

Comment:
You can probably just hard code the expectation here since you're passing in the name Betty Draper when creating the merchant. You shouldn't need to create a betty_merchant object. `expect(page).to have_content("Betty Draper")` would be a more concrete test as far as I understand.

Reply:
Made this change!